src/app/search/search.html

```diff
@@ -0,0 +1,26 @@
+<div class="gh-search well">
+    <form role="form">
+        <div class="form-group has-feedback">
+            <label class="control-label">Search: </label>
+            <input type="text" class="form-control" placeholder="Search"
+                   ng-model="searchCtrl.query"
+                   ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }"
+                   ng-change="searchCtrl.searchRepos()"/>
+            <i class="form-control-feedback fa fa-search"></i>
+        </div>
+    </form>
+
+    <ol>
+        <li ng-repeat="repo in searchCtrl.repos">
+            <span ng-bind="repo.full_name"></span>
```

Comment:
> Why ng-bind and not {{}}?

While your angularjs is bootstrapping, the user might see your placed brackets in the html. Changing from {{}} to strict ng-bind did save us like 20% in every ```scope.$digest```

More here http://stackoverflow.com/questions/16125872/angularjs-why-ng-bind-is-better-than-in-angular